ibis/backends/pandas/execution/selection.py

```diff
@@ -333,7 +352,8 @@ def build_df_from_selection(
             renamed_cols[from_col] = to_cols[0]
         else:
             for new_col in to_cols:
-                result[new_col] = result.loc[:, from_col]
```

Comment:
you need to copy the frame before here

Reply:
What is the material difference for users of ibis other than slower execution? Is there a concrete example showing the problem that involves this code path?

Reply:
you should never suppress these - they may change and the assignment is a bit janky - don't use .loc here as it's not idiomatic

Reply:
also fixing it properly will not involve a perf hit if u just filter

Reply:
> you should never suppress these - they may change and the assignment is a bit janky - don't use .loc here as it's not idiomatic

Perhaps you can suggest a change that is compatible with both dask and pandas then. This code path is shared needs to work with both without a deeper refactoring.

Reply:
```
result[to_col] = result[from_col]
```

Reply:
This is fixed.